app/bundles/NotificationBundle/Tests/Controller/PopupControllerTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\NotificationBundle\Tests\Controller;
+
+use Mautic\CoreBundle\Test\MauticMysqlTestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+final class PopupControllerTest extends MauticMysqlTestCase
+{
+    /**
+     * Smoke test to ensure the '/s/notifications' route loads.
+     */
+    public function testIndexRouteSuccessfullyLoads(): void
+    {
+        $this->client->request(Request::METHOD_GET, '/notification');
```

Comment:
I tried to open this link but it gives me 404 [https://8080-joshuaestes-mautic-py0t2rb6yrh.ws-eu85.gitpod.io/s/notification]

I think it should include s so it is it `notifications` instead of `notification`, I find it strange that this test successeded

Reply:
![Screenshot from 2023-02-03 10-35-37](https://user-images.githubusercontent.com/447419/216643668-e4d5d00a-289d-4de0-a6a9-d20a8f30933b.png)

This is the page, there shouldn't be an `s` in there.